crates/store/re_types_core/src/loggable_batch.rs

```diff
@@ -66,6 +72,68 @@ pub trait ComponentBatch: LoggableBatch {
             .with_descriptor_override(descriptor)
     }
 
+    /// Serializes the contents of this [`ComponentBatch`].
+    ///
+    /// Once serialized, the data is ready to be logged into Rerun via the [`AsComponents`] trait.
+    ///
+    /// # Fallibility
+    ///
+    /// There are very few ways in which serialization can fail, all of which are very rare to hit
```

Comment:
Still worth mentioning a few of the ways it can fail (e.g. overflowing an `i32`)